README.md

```diff
@@ -89,6 +91,7 @@ A curated list of tools and resources for Platform Engineering.
 - [OpenCost — open source cost monitoring tool for Kubernetes](https://github.com/opencost/opencost)
 - [Apache SkyWalking — Application Performance Monitoring](https://github.com/apache/skywalking)
 - [SigNoz- an open-source alternative to DataDog, NewRelic, etc.](https://github.com/signoz/signoz)
+- [kube-green - A K8s operator to reduce CO2 footprint of your clusters](https://kube-green.dev/)
```

Comment:
Please remove this link, since it's already in https://github.com/Green-Software-Foundation/awesome-green-software